expect/_matchers.ts

```diff
@@ -89,13 +89,15 @@ export function toBeCloseTo(
   if (context.isNot) {
     if (pass) {
       throw new AssertionError(
-        `Expected the value not to be close to ${expected} (using ${numDigits} digits), but it is`,
+        context.customMessage ??
+          `Expected the value not to be close to ${expected} (using ${numDigits} digits), but it is`,
```

Comment:
I think this part should be something like:

```ts
const defaultMessage = `Expected the value not to be close to ${expected} (using ${numDigits} digits), but it is`
throw new AssertionError(
  context.customMessage ? `${context.customMessage}: ${defaultMessage}` : defaultMessage
);
```

Vitest, for example, shows the custom message like the above.

In Vitest:
```ts
expect(42, "custom message").toEqual(43);
```
This assertion error shows the message like:
```
AssertionError: custom message: expected 42 to deeply equal 43

- Expected
+ Received

- 43
+ 42
```

Reply:
Updated

Reply:
Oh, I missed this message. Sorry for the delay!